dashboard/cron.py

```diff
@@ -48,11 +48,18 @@ def util_function(UDW_course_id, sql_string, mysql_table, table_identifier=None)
 
     # drop duplicates
     df.drop_duplicates(keep='first', inplace=True)
+    
+    logger.debug("Duplicates dropped")
```

Comment:
Not sure about the usage of the debug message here.

Reply:
I thought the issue was in the duplicate removal. @pushyamig commented that the SQL could possibly be improved. I could remove this message but it still was useful to see the dataframe before and after duplicates to rule that out as a problem.